hygraph-content-source/src/hygraph-api-client.ts

```diff
@@ -475,7 +513,7 @@ export class HygraphApiClient {
           }`;
 
         try {
-            return this.contentClient.request(gql).then((response) => { return response.createAsset; });
+            return this.contentClient.request(gql).then((response: any) => { return response.createAsset; });
```

Comment:
@arsenyg can we have full type for the Asset response like with other entries, instead of `any`.